shared/wallets/sep7-confirm/container.tsx

```diff
@@ -53,11 +53,18 @@ const Connected = Container.connect(mapStateToProps, mapDispatchToProps, (stateP
     memo,
     memoType,
     message,
-    operation,
     originDomain,
     recipient,
     summary,
   } = stateProps.sep7ConfirmInfo
+
+  const rawOp = stateProps.sep7ConfirmInfo.operation
+  const operation = rawOp === 'pay' ? ('pay' as const) : rawOp === 'tx' ? ('tx' as const) : ('' as const)
```

Comment:
`const operation = rawOp === 'pay' ? 'pay' : rawOp === 'tx' ? 'tx' : ''` this seems to work in the TS Playground